rotkehlchen/exchanges/bitmex.py

```diff
@@ -148,13 +148,15 @@ def _api_query(
         data = ''
         if not options:
             request_path = request_path_no_args
+            signature_path = request_path
         else:
             request_path = request_path_no_args + '?' + urlencode(options)
+            signature_path = request_path_no_args if path == 'user/wallet' else request_path
```

Comment:
Why introduce a new variable here in both branches?

Wouldn't just

`request_path = request_path_no_args + '?' + urlencode(options) if path != 'user/wallet` else request_path` do?

Reply:
Because for 'user/wallet' we still need the "?currency=XBt" in the url (according their API explorer, that I've used), which is composed later on at:
https://github.com/rotki/rotki/blob/1b94a88c94268e3204c0a65833bf96b69afc2aa2/rotkehlchen/exchanges/bitmex.py#L173

Besides if we ever extend the functionality with "?filter=" query string it should work.

Reply:
aha I see.  All right that makes sense.